pkg/registry/apps/plugins/register.go

```diff
@@ -60,9 +64,21 @@ func (p *PluginsAppInstaller) InstallAPIs(
 		Version:  pluginsv0alpha1.GroupVersion.Version,
 		Resource: pluginsv0alpha1.PluginMetaKind().Plural(),
 	}
+
 	replacedStorage := map[schema.GroupVersionResource]rest.Storage{
 		pluginMetaGVR: pluginsapp.NewPluginMetaStorage(request.GetNamespaceMapper(p.cfg)),
 	}
+
+	if r, ok := p.reg.(*registry.InMemory); ok {
```

Comment:
@wbrowne i was making an assumption here that the read only custom storage would be used if the registry was `InMemory`, and unified storage if not. thoughts?